clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -865,6 +886,8 @@ public void onAssignment(Set<TopicPartition> assignment) {
         private final TopicPartition partition;
         private final CompletedFetch completedFetch;
         private final Iterator<? extends RecordBatch> batches;
+        private final Set<Long> abortedPids;
+        private final Queue<FetchResponse.AbortedTransaction> abortedTransactions;
```

Comment:
The fact that we actually use a priority queue is significant to the correctness of the algorithm, so maybe we should use the more generic type?